.eslintrc.js

```diff
@@ -6,16 +6,16 @@ module.exports = {
   },
   extends: [
     '@nuxtjs/eslint-config-typescript',
-    'plugin:nuxt/recommended'
+    'plugin:vitest/recommended'
   ],
-  plugins: ['vitest'],
   rules: {
-    semi: ['warn', 'always'],
+    semi: ['error', 'always'],
```

Comment:
Missing semicolons cause some syntax highlight issues to Vue SFC. We better error them instead of warning.